packages/fxa-settings/src/lib/storage-utils.ts

```diff
@@ -57,6 +57,8 @@ export interface StoredAccountData {
   verified?: boolean;
   alertText?: string;
   displayName?: string;
+  verificationMethod?: string;
+  verificationReason?: string;
```

Comment:
That's correct, in content server it was just stored in the backbone state in the account model. I put it in local storage, because other account data from this gql query was being [stored there](https://github.com/mozilla/fxa/blob/cb13c51b5aa22ae698e06d4db5caf0a25087567f/packages/fxa-settings/src/pages/Signin/index.tsx#L192) and it made sense to keep this data together.  Do you see the local storage being an issue for some reason?

Here are the other options I considered:
- Reach router state
- Browser session storage
- URL query params

I didn't actually consider the apollo cache. Since it's part of the signin response and not queried directly, I wasn't quite sure how that would feel, but I'm open to suggestions. Ultimately, I don't have much an opinion on it though.

Reply:
Okay, GQL seems promising. For simplicity sake, I'm going with reach router for now.